test/e2e/apimachinery/table_conversion.go

```diff
@@ -145,7 +145,13 @@ var _ = SIGDescribe("Servers with support for Table transformation", func() {
 		e2elog.Logf("Table:\n%s", out)
 	})
 
-	ginkgo.It("should return a 406 for a backend which does not implement metadata", func() {
+	/*
+			    Release : v1.16
+				Testname: API metadata HTTP return
+				Description: Issue a HTTP request to the API.
+		        HTTP request MUST return a HTTP status code of 406.
+	*/
+	framework.ConformanceIt("should return a 406 for a backend which does not implement metadata", func() {
```

Comment:
before promoting this to conformance, I'd expect it to request `v=v1`

Reply:
@liggitt do you think this is a small enough change that we could include it in this PR?

Reply:
yes

Reply:
Deleted comment to re-draft, before I saw your response.

Original message:
> @liggitt do you think this is a small enough change that we could include it in this PR?